test/e2e/framework/get-kubemark-resource-usage.go

```diff
@@ -36,6 +37,7 @@ func getMasterUsageByPrefix(prefix string) (string, error) {
 	return sshResult.Stdout, nil
 }
 
+// GetKubemarkMasterComponentsResourceUsage returns resource usage of kubemark.
```

Comment:
tiny nit / suggestion:
```suggestion
// GetKubemarkMasterComponentsResourceUsage returns the resource usage of kubemark.
```
is this a map of pod name -> resource usage? it would be helpful if this comment clarified what the returned mapping is

Reply:
That is a nice idea.
The function gathers cpu and memory usage from `ps ax -o %cpu,rss,command` with grepping `kube` and use name from `kube-<name>` as a pod name.
I will add that on the comment.